src/styles/blocks/card.scss

```
@@ -0,0 +1,84 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  width: $card-width;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 0;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    line-height: 18px;
+    font-weight: 500;
+    margin-bottom: 4px;
+    margin-top: 0;
+    color: $black-text;
+  }
+
+  &__product-code {
+    color: $grey-text;
+    margin-bottom: 16px;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+  }
+
+  &__review {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__prise {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__prise-text {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__prise-number {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+  }
+
+  &__buy-link {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 40px;
+    font-weight: 700;
+    font-size: 14px;
+    text-transform: uppercase;
+    background: #00acdc;
+    border-radius: 5px;
+    text-decoration: none;
+    color: #fff;
+    margin-bottom: 28;
+  }
+
+  &__buy-link:hover {
+    background: #fff;
+    border: 1px solid #00acdc ;
+    color: #00acdc;
+  }
+}
```

```suggestion
    margin-bottom: 28;
     border: 1px solid #00acdc ;
  }

  &__buy-link:hover {
    background: #fff;
    color: #00acdc;
  }
}
```
Не вказуй border в hover

---

ще добавь до * box-sizing: border-box;